custom_connectors/oauth2/bim_360.rb

```diff
@@ -131,84 +130,64 @@
-            "project_type list in the <a href='https://forge.autodesk." \
-            "com/en/docs/bim360/v1/overview/parameters' target= '_blank'>" \
-            'Parameters</a> guide' },
-          { name: 'value', label: 'Monetary value' },
-          { name: 'currency', control_type: 'select',
-            pick_list: 'currency_list',
-            toggle_hint: 'Select currency',
-            toggle_field: {
-              name: 'currency',
-              type: 'string',
-              control_type: 'text',
-              label: 'Project type',
-              toggle_hint: 'Use custom value'
-            } },
-          { name: 'status', control_type: 'select',
-            pick_list: 'status_list',
-            toggle_hint: 'Select status',
-            toggle_field: {
-              name: 'status',
-              type: 'string',
-              control_type: 'text',
-              label: 'Status',
-              toggle_hint: 'Use custom value'
-            } },
-          { name: 'job_number' },
-          { name: 'address_line_1' },
-          { name: 'address_line_2' },
-          { name: 'city' },
-          { name: 'state_or_province' },
-          { name: 'postal_code' },
-          { name: 'country' },
-          { name: 'postal_code' },
-          { name: 'country' },
-          { name: 'business_unit_id' },
-          { name: 'timezone', hint: 'Refer to the preconfigured ' \
-          "project_type list in the <a href='https://forge.autodesk." \
-          "com/en/docs/bim360/v1/overview/parameters' target= '_blank'>" \
-          'Parameters</a> guide' },
-          { name: 'language', control_type: 'select', pick_list:
-            [%w[English en], %w[German de]] },
-          { name: 'construction_type', hint: 'Refer to the preconfigured ' \
-            "project_type list in the <a href='https://forge.autodesk." \
-            "com/en/docs/bim360/v1/overview/parameters' target= '_blank'>" \
-            'Parameters</a> guide' },
-          { name: 'contract_type', hint: 'Refer to the preconfigured ' \
-            "project_type list in the <a href='https://forge.autodesk." \
-            "com/en/docs/bim360/v1/overview/parameters' target= '_blank'>" \
-            'Parameters</a> guide' },
-          { name: 'last_sign_in', hint: 'Timestamp of the last sign in,' \
-            ' YYYY-MM-DDThh:mm:ss.sssZ format' }
+          { name: 'type' },
+          { name: 'attributes', type: 'object', properties: [
+            { name: 'name' }
+          ]},
+          { name: 'relationships', type: 'object', properties: [
+            { name: 'hub', type: 'object', properties: [
+              { name: 'data', type: 'object', properties: [
+                { name: 'id', label: 'Hub ID' }
+              ]}
+            ]},
+            { name: 'rootFolder', type: 'object', properties: [
+              { name: 'data', type: 'object', properties: [
+                { name: 'id', label: 'Root Folder ID' }
```

Comment:
Please use all labels starting with Upper case followed by all lower case unless specific fields like ID, URL, etc.
```suggestion
                { name: 'id', label: 'Root folder ID' }
```